zomes/chat/src/validation.rs

```diff
@@ -28,17 +27,25 @@ pub(crate) fn joining_code(element: Element) -> ExternResult<ValidateCallbackRes
                         return Ok(ValidateCallbackResult::Invalid(format!("Joining code invalid: unexpected author ({:?})", author)))
                     }
 
-                    let signature = mem_proof.signature().clone();
-                    if let ElementEntry::Present(entry) = mem_proof.entry() {
-                        let jcp = JoiningCodePayload::try_from(entry.clone())?;
+                    let _signature = mem_proof.signature().clone();
```

Comment:
Looks great, just wondering why do we need to read this signature?

Reply:
just lazy